.github/PULL_REQUEST_TEMPLATE/register_did_method.md

```diff
@@ -0,0 +1,47 @@
+## Register a DID Method
+
+### Required Registration Criteria
+
+Please confirm:
+
+- [ ] The DID Method specification defines the identifer syntax.
+- [ ] The DID Method specification defines the supported DID Operations.
+- [ ] The DID Method specification contains a Privacy and Security Considerations.
+
+### Optional Registration Criteria
+
+Consider ensuring:
+
+- [ ] The registration entry contains contact information.
+- [ ] The registration entry contains registry information.
+
+### Registration Process
+
+In order to register a New DID Method, you must add a json file to [./methods](./methods).
+
+Here is an example registration entry:
+
+```jsonc
+{
+  // These fields are required
+  "name": "example",
+  "status": "registered",
+  "specification": "https://w3c-ccg.github.io/did-spec/",
+  // These fields are optional
+  "contactName": "W3C Credentials Community Group",
+  "contactEmail": "",
+  "contactWebsite": "",
+  "verifiableDataRegistry": "DID Specification"
+}
+```
+
+Please ensure your pull request and entry pass the required validation rules.
+
+### Registration Review Process
+
+Your pull request will be reviewed by the registry editors.
+
+Changes regarding the required criteria may be requested.
+
+If there are no objections or changes requested, 
+your method will be registered after a minimum of 7 days.
```

Comment:
we could waffle about it and say "but it could take 30 days or more."

that would set expectations without committing editors to a timeline.

Reply:
If the editors feel that such a waffle is required, I could perhaps be convinced.  I do feel that there should be *some* commitment to a maximum, by which point the submitter should expect a "registered" or "denied because ..." action, such that they can try to correct whatever failing was found.  "Rejection implied by silence" doesn't make me feel good.

Reply:
that is a good point

Reply:
I'm fine w/ max of 30 days. They can start loudly complaining at that point and I'd personally feel that it was justified.